src/components/Tiles/Tile.js

```diff
@@ -1,17 +1,31 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
-
+import { misc } from 'js/i18n/definitions';
 import { tilePropTypes } from './proptypes';
 
-const Tile = ({ url, text, compact }) => (
-  <Link className={compact ? 'coa-Tile--compact' : 'coa-Tile'} to={url}>
-    <div className="coa-Tile__content">
-      <p className="coa-Tile__text">{text}</p>
-      <i className="material-icons">arrow_forward</i>
-    </div>
-  </Link>
-);
+import guideIcon from 'images/guide_icon.png';
+
+const Tile = ({ url, text, compact, __typename }) => {
+  console.log("misc :", misc)
+  let typename = "arrow_forward"
```

Comment:
typename as a name here confuses me, maybe typeIcon? I'm also struggling with names for it

Reply:
i came up with icontype above